src/pages/FrontendMentor/skDashboard/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { Switch } from 'antd'
+import React, { useState } from 'react'
+import { ThemeProvider } from 'styled-components'
+import {
+  DashboardHeader,
+  FollowersCount,
+  MainPage,
+  GlobalStyles,
+  lightTheme,
+  darkTheme,
+  OverviewHeading,
+  CardSection,
+  StyledDarkMode,
+  LineSeparator,
+} from './styles/skDashboard.styled'
+
+import SocialMediaCard from '../../../components/skDashboard/SocialMediaCard'
+import FacebookIcon from '../../../assets/skDashboard/icon-facebook.svg'
+import TwitterIcon from '../../../assets/skDashboard/icon-twitter.svg'
+import InstagramIcon from '../../../assets/skDashboard/icon-instagram.svg'
+import YoutubeIcon from '../../../assets/skDashboard/icon-youtube.svg'
+import SocialOverviewCard from '../../../components/skDashboard/SocialOverviewCard'
+
+interface MediaCard {
+  names: string
+  platform: string
+  icon: string
+  followerCount: string
+  increase: boolean
+  change: string
+}
+
+interface OverviewCard {
+  icon: string
+  statsName: string
+  stats: string
+  increase: boolean
+  change: string
+}
+
```

Comment:
if these are similar to the ones above, consider exporting from here so theres less duplicated code; import to use in the other files (if you are using redux, these should be in the types file instead)

Reply:
0a4783a